meta-oe/recipes-core/toybox/toybox_0.8.2.bb

```diff
@@ -3,21 +3,27 @@ HOMEPAGE = "http://www.landley.net/toybox/"
 DEPENDS = "attr virtual/crypt"
 
 LICENSE = "BSD-0-Clause"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=f0b8b3dd6431bcaa245da0a08bd0d511"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=78659a599b9325da368f2f1eb88f19c7"
```

Comment:
what change in License is causing this checksum to change ? Please add that info to commit message

Reply:
updated commit message